src/Computation.hs

```diff
@@ -92,6 +123,27 @@ computeSurjectionLines g n left right bottom top =
                        let top_right = (startx_l' right, starty_l' top) in
                        computeSurjection g n lower_left top_right
 
+
+-- Computes the bijection to lie inside a bounding box defined by the corners of a box 
+-- defined by four straight lines, assuming their lower/left coordinates come first. 
+-- Their intersections give the corners.
+computeBijectionLines :: (Autofloat a) => StdGen -> Integer 
+                                   -> Line' a -> Line' a -> Line' a -> Line' a -> ([Pt2 a], StdGen)
+computeBijectionLines g n left right bottom top = 
+                       let lower_left = (startx_l' left, starty_l' bottom) in
+                       let top_right = (startx_l' right, starty_l' top) in
+                       computeBijection g n lower_left top_right
+
+-- Computes the injection to lie inside a bounding box defined by the corners of a box 
+-- defined by four straight lines, assuming their lower/left coordinates come first. 
+-- Their intersections give the corners.
+computeInjectionLines :: (Autofloat a) => StdGen -> Integer 
```

Comment:
Some code duplication here (the `Lines` functions). Could be refactored into a single `computeFunctionLines` function.

Reply:
I think that it is better like that since the name of the function is clearer (the other option is to ask the style writer to pass a parameter which specify the desired function style)  and the compaction is insignificant